spec/StringClassSpec.js

```diff
@@ -79,21 +78,20 @@ describe('test isQuestion method', () => {
 
   it('should return false if the string is not a question', () => {
     expect('TBNRTY'.isQuestion()).toBeFalse();
-    expect('tbnrty'.isQuestion()).toBeFalse();
-    expect('245Gy'.isQuestion()).toBeFalse();
+    expect('How are you doing? I\'m great.'.isQuestion()).toBeFalse();
```

Comment:
**Code Quality - Critical**

Add a test for a string that has a question mark within it and not at the end.